# joint state
# vrednosti za premikanje posameznega sklepa
#JointStateType type
int32 type
float32 position
float32 goal
float32 speed
